content/en/docs/tutorials/security/ns-level-pss.md

```diff
@@ -0,0 +1,160 @@
+
+```shell
+kubectl create ns example
+```
+The output is similar to this:
+```
+namespace/example created
+```
+
+# Applying one Pod Security Standard
+
+Enable Pod Security Standards on this namespace using labels supported by
+built-in Pod Security Admission. In this step we will warn on baseline pod
+security standard as per the latest version (default value)
+
+```shell
+kubectl label --overwrite ns example \
+  pod-security.kubernetes.io/warn=baseline \
+  pod-security.kubernetes.io/warn-version=latest
+```
+
+# Applying multiple Pod Security Standards
+
+Multiple pod security standards can be enabled on any namespace, using labels.
+Following command will `enforce` the `baseline` Pod Security Standard, but
+`warn` and `audit` for `restricted` Pod Security Standards as per the latest
+version (default value)
+
+```
+kubectl label --overwrite ns example \
+  pod-security.kubernetes.io/enforce=baseline \
+  pod-security.kubernetes.io/enforce-version=latest \
+  pod-security.kubernetes.io/warn=restricted \
+  pod-security.kubernetes.io/warn-version=latest \
+  pod-security.kubernetes.io/audit=restricted \
+  pod-security.kubernetes.io/audit-version=latest
+```
+
+# Create Pod
+
+1. Create a minimal pod in `example` namespace:
+
+    ```shell
+    cat <<EOF > /tmp/pss/nginx-pod.yaml
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      name: nginx
+    spec:
+      containers:
+        - image: nginx
+          name: nginx
+          ports:
+            - containerPort: 80
+    EOF
+    ```
+2. Apply the pod spec to the cluster in `example` namespace:
+    ```shell
+    kubectl apply -n example -f /tmp/pss/nginx-pod.yaml
+    ```
+    The output is similar to this:
+    ```
+    Warning: would violate PodSecurity "restricted:latest": allowPrivilegeEscalation != false (container "nginx" must set securityContext.allowPrivilegeEscalation=false), unrestricted capabilities (container "nginx" must set securityContext.capabilities.drop=["ALL"]), runAsNonRoot != true (pod or container "nginx" must set securityContext.runAsNonRoot=true), seccompProfile (pod or container "nginx" must set securityContext.seccompProfile.type to "RuntimeDefault" or "Localhost")
+    pod/nginx created
+    ```
+   
+3. Apply the pod spec to the cluster in `default` namespace:
+    ```shell
+     kubectl apply -n default -f /tmp/pss/nginx-pod.yaml
+    ```
+    Output is similar to this:
+    ```
+    pod/nginx created
+    ```
+
+As you can see the Pod Security Standards were applied only to `example` 
+namespace. For `default` namespace, pod was created without any warnings. 
+To apply pod security standards to multiple namespaces at once at cluster 
+level, please 
+[follow this tutorial](/docs/tutorials/security/cluster-level-pss).
+
+# Clean up
+
+Run `kind delete cluster -name psa-ns-level` to delete the cluster created.
+
+## {{% heading "whatsnext" %}}
+
+- Run a
+[gist](https://gist.github.com/PushkarJ/c694bac35c2d100f906861667474afb5)
```

Comment:
Please move this from a gist into an example you commit (into `content/en/examples`), or omit the advice. It's OK to link to the example script on GitHub if that's useful to readers.
We definitely prefer not to recommend running 3rd party code from gists we don't control.

Reply:
Done